lib/utils.js

```diff
@@ -281,14 +281,25 @@ exports.createAssertionCheck = function (assertions, unexpectedGlobalAssertion,
             const isGlobal = isGlobalAssertion(node.callee);
             const assertion = isGlobal ? node.callee.name : node.callee.property.name;
 
-            context.report({
-                node: node,
-                messageId: isGlobalAssertion(node.callee) ? unexpectedGlobalAssertion : unexpectedLocalAssertion,
+            const reportErrorObject = {
+                node,
                 data: {
                     assertVar,
                     assertion
                 }
-            });
+            };
+            const errorMessageConfigForAssertion = errorMessageConfig[assertion];
+            /* istanbul ignore else: correctly does nothing */
+            if (errorMessageConfigForAssertion) {
+                /* istanbul ignore else: correctly does nothing */
```

Comment:
The `else` paths can be properly tested once [the rule mentioned in this comment](https://github.com/platinumazure/eslint-plugin-qunit/pull/80#issuecomment-633150211) is implemented.